### PR TITLE
feature(ui-ux): update borrow loan to v2

### PR DESCRIPTION
**Pull request by kyleleow**
#### What this PR does / why we need it:
- Update borrow loan to v2

#### Which issue(s) does this PR fixes?:

Fixes #

#### Additional comments?:
- Will add translation and e2e in separate Pr

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [x] No console errors on web
- [x] Tested on Light mode and Dark mode*
- [x] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [ ] Added e2e tests*
- [ ] Added translations*

### CODE REVIEW

**Review by fullstackninja864 — CHANGES_REQUESTED** [review]
Not able to borrow 2nd loan token.

https://user-images.githubusercontent.com/53080940/195676808-a0c6a704-d3dd-4f3f-9c35-e61669ad2cc8.MP4

File: mobile-app/app/screens/AppNavigator/screens/Loans/components/BottomSheetVaultList.tsx [hunk 1]
```diff
@@ -53,120 +50,123 @@ export const BottomSheetVaultList = ({
...
+                  value={item.loanScheme.interestRate}
+                  thousandSeparator
+                  displayType="text"
+                  renderText={(value) => (
+                    <ThemedTextV2
+                      style={tailwind("text-xs font-normal-v2")}
+                      light={tailwind("text-mono-light-v2-700")}
+                      dark={tailwind("text-mono-dark-v2-700")}
+                    >
+                      {translate("", "{{value}}% interest", { value })}
```
[the remark this answers is not in the document]
Comment (kyleleow) [reply mobile-app/app/screens/AppNavigator/screens/Loans/components/BottomSheetVaultList.tsx:116]: we will go with lowercase

File: mobile-app/app/screens/AppNavigator/screens/Loans/hooks/DFIRequirementForDusdLoanAndCollateral.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+ *
+ * @returns
+ */
+export function useDFIRequirementForDusdLoanAndCollateral(
+  props: useDFIRequirementForDusdLoanAndCollateralProps
+) {
+  const collateralTokens: CollateralToken[] = useSelector(
+    (state: RootState) => state.loans.collateralTokens
+  );
+  const hasDUSDLoan = props.loanAmounts.some(
```
Comment (lykalabrada) [inline mobile-app/app/screens/AppNavigator/screens/Loans/hooks/DFIRequirementForDusdLoanAndCollateral.ts:35]: `hasDUSDLoan` is unused

File: mobile-app/app/screens/AppNavigator/screens/Loans/hooks/DFIRequirementForDusdLoanAndCollateral.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    dfiCollateralToken?.factor,
+    "ACTIVE",
+    "COLLATERAL"
+  );
+  const dfiCollateralValue = new BigNumber(dfiActivePrice).multipliedBy(
+    props.collateralAmounts.find((col) => col.displaySymbol === "DFI")
+      ?.amount ?? 0
+  );
+  const isDFILessThanHalfOfRequiredCollateral = dfiCollateralValue.isLessThan(
+    new BigNumber(props.loanValue)
```
Comment (lykalabrada) [inline mobile-app/app/screens/AppNavigator/screens/Loans/hooks/DFIRequirementForDusdLoanAndCollateral.ts:55]: I will also add this same logic for Remove collateral, so just want to align:
1. This rule should only apply if vault has DUSD collateral (and borrowing DUSD)
2. As long as DUSD `loanValue` and collateral is greater than zero, 50%DFI is required. User shouldn't be able to take DUSD loan at all

This is based on this [this article](https://blog.defichain.com/new-rules-guide-decentralized-loans/) 🤔

Comment (kyleleow) [reply mobile-app/app/screens/AppNavigator/screens/Loans/hooks/DFIRequirementForDusdLoanAndCollateral.ts:55]: 1. For a vault that has taken any loan, it will have a certain amount of DFI and/or DUSD as collateral, because that is the basic requirement to take loan. Any other collateral added will not increase amount of loan you can take if 50% of the total collateral to satisfy min col ratio is not DFI and/or DUSD
2. The new rule is actually trying to limit DUSD as being the majority of the collateral IF the vault is trying to take DUSD loan. Instead of allowing any ratio of DFI and DUSD as collateral, we now force it to have the original requirement on my previous point to only include DFI to satisfy the min col ratio.

tldr: the reason i didnt include checking for DUSD collateral because checking for DFI collateral is enough to decide
